src/main/java/org/springframework/data/elasticsearch/client/reactive/DefaultReactiveElasticsearchClient.java

```diff
@@ -674,6 +677,11 @@ private RequestBodySpec sendRequest(WebClient webClient, String logId, Request r
 		return sendRequest(getMappingsRequest, requestCreator.getMapping(), GetMappingsResponse.class, headers).next();
 	}
 
+	@Override
+	public Mono<GetFieldMappingsResponse> getFieldMapping(final HttpHeaders headers, final GetFieldMappingsRequest getFieldMappingsRequest) {
```

Comment:
```suggestion
	public Mono<GetFieldMappingsResponse> getFieldMapping(HttpHeaders headers, GetFieldMappingsRequest getFieldMappingsRequest) {
```

we don't use `final`on the parameters

Reply:
Fixed.